code/modules/admin/verbs/playsound.dm

```diff
@@ -6,7 +6,7 @@ var/list/sounds_cache = list()
 	if(!check_rights(R_SOUNDS))
 		return
 
-	var/sound/awful_sound = sound(null, repeat = 0, wait = 0, channel = 777)
+	var/sound/awful_sound = sound(null, repeat = 0, wait = 0, channel = CHANNEL_ADMIN)

```

Comment:
the reason this doesn't use the new proc is so you're not creating a new null sound for everyone in the player list.